packages/@lwc/integration-karma/test/mixed-shadow-mode/retargeting/x/syntheticParent/syntheticParent.js

```diff
@@ -0,0 +1,19 @@
+import { LightningElement } from 'lwc';
+
+if (!customElements.get('mixed-shadow-mode-retargeting')) {
+    customElements.define(
+        'mixed-shadow-mode-retargeting',
+        class extends HTMLElement {
+            constructor() {
+                super();
+                this._shadowRoot = this.attachShadow({ mode: 'closed' });
+                this._shadowRoot.innerHTML = `<slot></slot><button></button>`;
```

Comment:
This `<button>` is never clicked right? What happens if it is clicked?

Reply:
It is clicked via the public `click()` method a few lines below in the `when event dispatched on an element inside a native shadow root` test case. `syntheticParent.shadowRoot.querySelector('mixed-shadow-mode-retargeting').click()`.